### PR TITLE
Added getLoaderAlert() method and getAllCreatedWorkspacesNames() into page objects.

### PR DESCRIPTION
**Pull request by nallikaea**
### What does this PR do?
New methods getLoaderAlert() and getAllCreatedWorkspacesNames() is used in RefusedOAuthFactory.spec.ts e2e-codready test suite.

- getLoaderAlert() - get alert message if any error on workspace loader page;
- getAllCreatedWorkspacesNames() returns names of created workspaces as string array;

### What issues does this PR fix or reference?

as part of: 

- https://github.com/eclipse/che/issues/22001

- https://issues.redhat.com/browse/CRW-3816

### How to test this PR?

Test run: 

image

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix or new feature ](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix-or-new-feature)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: tests/e2e/pageobjects/dashboard/Workspaces.ts [hunk 1]
```diff
@@ -192,4 +211,7 @@ export class Workspaces {
         return By.xpath(`${this.getWorkspaceListItemLocator(workspaceName)}//td[@data-key=5]//a[text()='Open']`);
     }
 
+    private getWorkspaceItemNameTableSectionLocator() {
```
Comment (ScrewTSW) [inline tests/e2e/pageobjects/dashboard/Workspaces.ts]: Please consider changing this to a constant

**Review by ScrewTSW — APPROVED** [review]
LGTM, just one small remark

File: tests/e2e/pageobjects/dashboard/Workspaces.ts (hunk 1 above)
Comment (nallikaea) [reply tests/e2e/pageobjects/dashboard/Workspaces.ts]: thnx, done

**Review by ScrewTSW — APPROVED** [review]
LGTM

**Review by dmytro-ndp — APPROVED** [review]
LGTM

**Pull request merged**